simple_history/models.py

```diff
@@ -222,6 +223,22 @@ def get_instance(self):
                 for field in fields.values()
             })
 
+        def get_next_record(self):
+            """
+            Get the next history record for the instance. `None` if last.
+            """
+            return self.instance.history.filter(
+                Q(history_date__gt=self.history_date)
+            ).order_by('history_date').first()
+
+        def get_prev_record(self):
+            """
+            Get the previous history record for the instance. `None` if first.
+            """
+            return self.instance.history.filter(
+                Q(history_date__lt=self.history_date)
+            ).order_by('history_date').first()
```

Comment:
This should reverse order by history date or take `.last()` with the existing order. We should also use say 3 records in `test_get_prev_record` / `test_get_next_record` and get the previous and next records of the last / first records, respectively, to ensure we catch this type of bug in the unit tests.

Reply:
Good catch @ncvc

Reply:
very good catch, thanks!